spk/synocli-net/Makefile

```
@@ -26,10 +26,10 @@ OPTIONAL_DESC := $(OPTIONAL_DESC)", ser2net"
 endif
 
 MAINTAINER = ymartin59
-DESCRIPTION = "SynoCli Network Tools provides a set of small command-line utilities: screen, tmux, mosh, socat, nmap, sshfs, rsync$(OPTIONAL_DESC). Credits to Sebastian Schmidt \(publicarray\) for icons"
+DESCRIPTION = "SynoCli Network Tools provides a set of small command-line utilities: screen, tmux, mosh, socat, nmap, sshfs, rsync, autossh$(OPTIONAL_DESC). Credits to Sebastian Schmidt \(publicarray\) for icons"
 DISPLAY_NAME = SynoCli Network Tools
 STARTABLE = no
-CHANGELOG = "1. Add rsync<br/>2. Add ser2net"
+CHANGELOG = "1. Add rsync<br/>2. Add ser2net<br/>2. Add autossh"
```

CHANGELOG resets at each package revision
```suggestion
CHANGELOG = "1. Add autossh"
```